src/status_im/ui/screens/communities/request_to_join_bottom_sheet_redesign.cljs

```diff
@@ -0,0 +1,140 @@
+                       :content "Don't send a lot of small messages right after each other. Do not disrupt chat by spamming."}
+                      {:index 4
+                       :title "No pornographic, adult or NSFW material"
+                       :content "This is a community server and not meant to share this kind of material."}
+                      {:index 5
+                       :title "No advertisements"
+                       :content "We do not tolerate any kind of advertisements, whether it be for other communities or streams."}
+                      {:index 6
+                       :title "No offensive names and profile pictures"
+                       :content "You will be asked to change your name or picture if the staff deems them inappropriate."}])
+
+(defn community-rule-item [{:keys [title content index]}]
+  [react/view
+   {:style {:flex 1 :margin-top 16}}
+   [react/view
+    {:style
+     {:flex 1
+      :flex-direction :row
+      :align-items :center}}
+    [react/view
+     {:style
+      {:height 18
+       :width 18
+       :margin-left 1
+       :margin-right 9
+       :background-color quo2.colors/white
+       :border-color quo2.colors/neutral-20
+       :border-width 1
+       :border-radius 6}}
+     [quo2.text/text {:style
+                      {:margin-left :auto
+                       :margin-right :auto
+                       :margin-top :auto
+                       :margin-bottom :auto}
+                      :accessibility-label :communities-rule-index
+                      :weight              :medium
+                      :size                :label}
+
+      (str index)]]
+    [quo2.text/text
+     {:accessibility-label :communities-rule-title
+      :weight              :semi-bold
+      :size                :paragraph-2}
+     title]]
+   [quo2.text/text
+    {:style {:margin-left 28}
+     :accessibility-label :communities-rule-content
+     :size :paragraph-2}
+    content]])
+
+(defn community-rules-list [rules]
+
+  [list/flat-list
+   {:shows-horizontal-scroll-indicator false
+    :data                              rules
+    :render-fn                         community-rule-item}])
+
+(defn rules-checkbox [checkbox-state on-value-change]
+  [react/view {:style
+               {:flex 1
+                :flex-direction :row
+                :margin-top 20
+                :background-color quo2.colors/neutral-20
+                :padding 12
+                :border-radius 12
+                :border-width 1
+                :border-color quo2.colors/neutral-30}}
+   [quo/checkbox {:value       @checkbox-state
+                  :on-change on-value-change}]
+   [quo2.text/text {:accessibility-label :communities-accept-rules
+                    :size                :paragraph-2
+                    :style {:margin-left 8}}
+    (i18n/label :t/accept-community-rules)]])
+
+(defn request-to-join [community]
+  (let [checkbox-state (reagent/atom false)]
```

Comment:
Might be better to make it `agreed-to-terms?` or `agreed?` since it's a boolean we add a question mark at the end .. And I think it's common for atom's name to describe what it's storing.

Reply:
will adjust!